KMaina/Fast-Food-App
# #160298217 Page to Add a Food Item

#### What does this PR do?
Allow an admin to add a food item
#### Description of Task to be completed?
An admin should be able to

- Upload an image of the food item to be added

- Give a short description in the textarea

#### How should this be manually tested?

- Clone the repo onto your local machine.

- Open a terminal and run `cd Fast-Food-App`

- run `git checkout -b ft-admin-food-add-160298197`

- run `git pull https://github.com/KMaina/Fast-Food-App.git ft-admin-food-add-160298197`

- In the folder where you cloned the repo, you should open the UI folder

- Open the admin-add-food.html to see it in your browser for review

#### Any background context you want to provide?
This is simply the UI design, no functionality has been built in yet.
#### What are the relevant pivotal tracker stories?
#160298217
